nixos/modules/services/networking/hylafax/systemd.nix

```diff
@@ -7,12 +7,30 @@
 
 let
 
+  inherit (lib.cli) toGNUCommandLine;
+  inherit (lib) optional;
   inherit (lib) mkIf mkMerge;
-  inherit (lib) concatStringsSep optionalString;
+  inherit (lib)
+    concatLines
+    concatStringsSep
+    escapeShellArgs
+    optionalString
+    ;
 
   cfg = config.services.hylafax;
   mapModems = lib.forEach (lib.attrValues cfg.modems);
 
+  mkSpoolCmd =
+    prefix: program: posArg: options:
+    let
+      start = "${prefix}${pkgs.hylafaxplus}/spool/bin/${program}";
```

Comment:
Do we have no package option here?

Reply:
No, [this was discussed and rejected](https://github.com/NixOS/nixpkgs/pull/44496#pullrequestreview-143413521)

Reply:
I think the opinion about that has changed a lot since 2018. I think generally almost every module now has a package option which is often useful for various things like pinning older versions, applying patches or applying pre compression to static assets.

Reply:
I've added a `package` option in a separate commit.